src/transaction/internals.rs

```diff
@@ -0,0 +1,54 @@
+use crate::{Input, Output, Transaction, Witness};
+
+#[cfg(feature = "internals")]
```

Comment:
Does this really need to be under a feature flag? I feel like we should be more opinionated about what are standard public apis are for to reduce feature flag complexity.

Reply:
This `internals` flag is used for methods that shouldn't be exposed by the API. I think we should avoid creating many functions that shouldn't be called by the user. The inputs/outputs/witnesses are coupled and should be manipulated via the API of the builder and of the transaction itself

Reply:
It looks like this feature flag isn't used at all anywhere, should we just remove this?

Reply:
I think @digorithm was using that for some test purposes

Reply:
In that case, the only usage is for integrated tests purposes: https://github.com/FuelLabs/fuel-tx/blob/a77f4e84eb40e75a806e03c41ce44285b6a18bdc/test-helpers/Cargo.toml#L9

Reply:
maybe we could consolidate these flags under `test-helpers` or something instead of making a bunch of different flags for things that are just dev-deps?